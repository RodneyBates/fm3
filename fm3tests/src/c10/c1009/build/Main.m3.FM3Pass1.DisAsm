 ItkEOF()
 ItkRightEnd()
 ItkModuleLtPatch(C4,L2(16_2),[9,0])
 ItkModuleBeginPatch(C57,L2(16_2),[17,2])
 ItkModuleRt(L2(16_2),[9,0])
 ItkDeclScopeRt(L1(16_1))
 ItkOpenScopeRt(L1(16_1))
 ItkStmtListRt(L0(16_0),[18,2])
 ItkStmtListLt(L0(16_0),[18,2])
 ItkOpenDeclListLtPatch(C10,L0(16_0),[15,2])
 ItkOpenDeclListRt(L0(16_0),[15,2])
 ItkSkipLtPatch(C10,L1(16_1))
 ItkSkipRt(L1(16_1))
 ItkDeclValAbsent([17,0])
 ItkVarDeclValue([15,11])
 ItkReservedIdRef(Rid"CHAR",[15,13])
 ItkVarDeclType([15,6])
 ItkVarDeclIdListLtPatch(C10,L0(16_0),[15,6])
 ItkVarDeclIdListRt(L0(16_0),[15,6])
 ItkOpenScopeLt(L1(16_1))
 ItkDeclScopeLt(L1(16_1))
 ItkLeftEnd()
 ItkBOF()
     0
