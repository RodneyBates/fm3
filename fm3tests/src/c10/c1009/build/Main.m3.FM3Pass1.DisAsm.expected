 ItkEOF()
 ItkRightEnd()
 ItkModuleLtPatch(C4,L1(16_1),[9,0])
 ItkModuleBeginPatch(C45,L1(16_1),[17,2])
 ItkModuleRt(L1(16_1),[9,0])
 ItkDeclScopeRt(L1(16_1))
 ItkLookupScopeRt(L1(16_1))
 ItkStmtListLt(L0(16_0),[18,2])
 ItkStmtListRt(L0(16_0),[18,2])
 ItkBlockDeclListLtPatch(C10,L0(16_0),[15,2])
 ItkBlockDeclListRt(L0(16_0),[15,2])
 ItkSkipLtPatch(C10,L1(16_1))
 ItkSkipRt(L1(16_1))
 ItkReservedId(Rid"CHAR",[15,13])
 ItkVarDeclIdListLtPatch(C10,L0(16_0),[15,6])
 ItkVarDeclIdListRt(L0(16_0),[15,6])
 ItkLookupScopeLt(L1(16_1))
 ItkDeclScopeLt(L1(16_1))
 ItkLeftEnd()
 ItkBOF()
     0
